### ISSUE TITLE
--version outputs wrong version

### ISSUE THREAD

**Issue by silverwind** [sindresorhus/xo#256]
Not sure what's going on here
````console
$ npm i -g xo@0.18.2
/Users/silverwind/.npm-global/bin/xo -> /Users/silverwind/.npm-global/lib/node_modules/xo/cli.js
$ jq -r .version /Users/silverwind/.npm-global/lib/node_modules/xo/package.json
0.18.2
$ /Users/silverwind/.npm-global/lib/node_modules/xo/cli.js --version
0.19.0
````

**Comment by silverwind** [comment]
Resolved after reinstalling the module, weird.

**Comment by silverwind** [comment]
Looks like there actually is a published `0.19.0`:

```
     "0.18.0": "2017-03-18T10:02:17.631Z",
     "0.19.0": "2017-03-25T20:05:30.063Z",
     "0.18.1": "2017-03-25T20:13:39.475Z",
     "0.18.2": "2017-05-04T02:42:21.127Z"
```
